linkerd/app/inbound/src/lib.rs

```diff
@@ -351,15 +343,13 @@ impl Config {
         ));
 
         let http_server = svc::stack(http_router)
-            // Limits the amount of time each request waits to obtain a
-            // ready service.
-            .push_timeout(dispatch_timeout)
```

Comment:
This should probably become a failfast, but we already have a failfast on the http router...